client/i18n/locales/english/translations.json

```
@@ -99,6 +99,7 @@
     "private-name": "Your name will not appear on your certifications, if this is set to private.",
     "claim-legacy": "Once you've earned the following freeCodeCamp certifications, you'll be able to claim the {{cert}}:",
     "for": "Account Settings for {{username}}",
+    "sound-mode": "Campfire Mode adds the pleasant sound of an acoustic guitar throughout the website. You'll get musical feedback when you type in the editor, complete challenges, claim certifications, and more.",
```

```suggestion
    "sound-mode": "This adds the pleasant sound of acoustic guitar throughout the website. You'll get musical feedback as you type in the editor, complete challenges, claim certifications, and more.",
```
I'm not saying this is better, but it is shorter!